pypesto/optimize/optimizer.py

```diff
@@ -253,10 +253,12 @@ class ScipyOptimizer(Optimizer):
     """
     Use the SciPy optimizers.
 
-    Find details on the optimizer and configuration options at:
-    https://docs.scipy.org/doc/scipy/reference/generated/scipy.\
-        optimize.minimize.html#scipy.optimize.minimize
-    """
+    Find details on the optimizer and configuration options at the `Scipy
+    documentation <https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html#scipy.optimize.minimize>`_.
+
+    .. warning:: Least squares optimizers may face errors in case of non
+    continuous differentiable objective functions (e.g. Laplace priors)
```

Comment:
```suggestion
    .. warning:: Least-squares optimizers may raise exceptions in case of 
    non-continuously differentiable objective functions (e.g. Laplace priors).
```